schemas/cloud-gateway-schema.json

```
@@ -32,6 +32,80 @@
                                                 }
                                             }
                                         },
+                                        "security" :{
+                                            "type": "object",
+                                            "description": "Security settings",
+                                            "properties": {
+                                                "oauth2": {
+                                                    "type": "object",
+                                                    "description": "Enable OAuth2 login flow",
+                                                    "properties": {
+                                                        "client": {
+                                                            "type": "object",
+                                                            "description": "OAuth2 client settings",
+                                                            "properties": {
+                                                                "registration": {
+                                                                    "type": "object",
+                                                                    "description": "OAuth2 client registration settings",
+                                                                    "patternProperties": {
+                                                                        "$*.^": {
+                                                                            "type": "object",
+                                                                            "description": "OAuth2 provider-specific settings",
+                                                                            "properties": {
+                                                                                "issuer": {
+                                                                                    "type": "string",
+                                                                                    "description": ""
```

```suggestion
                                                                                    "description": "The URI used to discover the configuration information for an OAuth 2.0 or OpenID Connect 1.0 Provider."
```
I am not sure, bu tI guess this value is not used